src/site/stages/build/process-cms-exports/schemas/transformed/node-health_care_region_detail_page.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-health_care_region_detail_page'] },
+    entityType: { enum: ['node'] },
+    entityBundle: { enum: ['health_care_region_detail_page'] },
+    title: { type: 'string' },
+    changed: { type: 'number' },
+    entityUrl: {
+      type: 'object',
+      // TODO: Add breadcrumb here
+      properties: {
+        path: { type: 'string' },
+      },
+      required: ['path'],
+    },
+    fieldAlert: { type: ['string', 'null'] },
+    fieldContentBlock: {
+      type: 'array',
+      items: { $ref: 'Paragraph' },
+    },
+    fieldFeaturedContent: { type: 'array' },
```

Comment:
Of anything in particular that you could tell?

Reply:
:man_facepalming: I meant to fix this before I marked it ready - thank you.